### ISSUE TITLE
Errors on checking for update with Sabnzbd 3.0

### ISSUE THREAD

**Issue by Dazog** [sabnzbd/sabnzbd#1432]
These error's appeared when a new update became avaliable:

```
Traceback (most recent call last):  File "lib\site-packages\cherrypy\_cprequest.py", line 638, in respond  File "lib\site-packages\cherrypy\_cprequest.py", line 697, in _do_respond  File "lib\site-packages\cherrypy\lib\encoding.py", line 219, in __call__  File "lib\site-packages\cherrypy\_cpdispatch.py", line 54, in __call__  File "sabnzbd\interface.py", line 435, in tapi  File "sabnzbd\api.py", line 99, in api_handler  File "sabnzbd\api.py", line 452, in _api_fullstatus  File "sabnzbd\api.py", line 981, in report  File "json\__init__.py", line 231, in dumps  File "json\encoder.py", line 199, in encode  File "json\encoder.py", line 257, in iterencode  File "json\encoder.py", line 179, in defaultTypeError: Object of type bytes is not JSON serializable
--
ERROR 3 minutes ago [08/May/2020:10:34:48] HTTP Traceback (most recent call last):  File "lib\site-packages\cherrypy\_cprequest.py", line 638, in respond  File "lib\site-packages\cherrypy\_cprequest.py", line 697, in _do_respond  File "lib\site-packages\cherrypy\lib\encoding.py", line 219, in __call__  File "lib\site-packages\cherrypy\_cpdispatch.py", line 54, in __call__  File "sabnzbd\interface.py", line 435, in tapi  File "sabnzbd\api.py", line 99, in api_handler  File "sabnzbd\api.py", line 452, in _api_fullstatus  File "sabnzbd\api.py", line 981, in report  File "json\__init__.py", line 231, in dumps  File "json\encoder.py", line 199, in encode  File "json\encoder.py", line 257, in iterencode  File "json\encoder.py", line 179, in defaultTypeError: Object of type bytes is not JSON serializable
ERROR 3 minutes ago [08/May/2020:10:34:38] HTTP Traceback (most recent call last):  File "lib\site-packages\cherrypy\_cprequest.py", line 638, in respond  File "lib\site-packages\cherrypy\_cprequest.py", line 697, in _do_respond  File "lib\site-packages\cherrypy\lib\encoding.py", line 219, in __call__  File "lib\site-packages\cherrypy\_cpdispatch.py", line 54, in __call__  File "sabnzbd\interface.py", line 435, in tapi  File "sabnzbd\api.py", line 99, in api_handler  File "sabnzbd\api.py", line 452, in _api_fullstatus  File "sabnzbd\api.py", line 981, in report  File "json\__init__.py", line 231, in dumps  File "json\encoder.py", line 199, in encode  File "json\encoder.py", line 257, in iterencode  File "json\encoder.py", line 179, in defaultTypeError: Object of type bytes is not JSON serializable
INFO Update Available! b'3.0.0Alpha2'
```

**Comment by thezoggy** [comment]
I can confirm I see this as well
```
2020-05-08 11:42:58,157::DEBUG::[misc:371] Checked for a new release, cur= 3000001, latest= 2030999 (on https://sabnzbd.org/downloads.html), latest_test= 3000002 (on https://sabnzbd.org/downloads.html)
2020-05-08 11:43:05,987::ERROR::[_cplogging:213] [08/May/2020:11:43:05] HTTP 
Traceback (most recent call last):
  File "lib\site-packages\cherrypy\_cprequest.py", line 638, in respond
  File "lib\site-packages\cherrypy\_cprequest.py", line 697, in _do_respond
  File "lib\site-packages\cherrypy\lib\encoding.py", line 219, in __call__
  File "lib\site-packages\cherrypy\_cpdispatch.py", line 54, in __call__
  File "sabnzbd\interface.py", line 435, in tapi
  File "sabnzbd\api.py", line 99, in api_handler
  File "sabnzbd\api.py", line 452, in _api_fullstatus
  File "sabnzbd\api.py", line 981, in report
  File "json\__init__.py", line 231, in dumps
  File "json\encoder.py", line 199, in encode
  File "json\encoder.py", line 257, in iterencode
  File "json\encoder.py", line 179, in default
TypeError: Object of type bytes is not JSON serializable
2020-05-08 11:43:05,987::INFO::[_cplogging:213] [08/May/2020:11:43:05] HTTP 
```

**Comment by Safihre** [comment]
Is this on Alpha1 or Alpha2?

**Comment by Safihre** [comment]
Aah, this was in Alpha1 but is fixed in Alpha1 via efcb09a5b7555a87ab5b44a803360663a20f6e45
